lib/media/track/localvideotrack.js

```diff
@@ -62,6 +62,14 @@ class LocalVideoTrack extends LocalMediaVideoTrack {
     return `[LocalVideoTrack #${this._instanceId}: ${this.id}]`;
   }
 
+  /**
+   * @private
+   */
+  _canCaptureFrames() {
+    const isPublishing = !!this._trackSender._clones.size;
```

Comment:
Can we have a public `isPublishing` member on `_trackSender` instead of trying to access the private member `_clones`?